client/src/app/core/core-services/active-meeting.service.ts

```diff
@@ -1,14 +1,42 @@
 import { Injectable } from '@angular/core';
 
-import { BehaviorSubject, Observable } from 'rxjs';
+import { BehaviorSubject, Observable, Subscription } from 'rxjs';
+
+import { Meeting } from 'app/shared/models/event-management/meeting';
+import { ViewMeeting } from 'app/site/event-management/models/view-meeting';
+import { AutoupdateService, ModelSubscription } from './autoupdate.service';
+import { MeetingRepositoryService } from '../repositories/event-management/meeting-repository.service';
+import { DEFAULT_FIELDSET, SimplifiedModelRequest } from './model-request-builder.service';
 
 @Injectable({
     providedIn: 'root'
 })
 export class ActiveMeetingService {
     private meetingIdSubject = new BehaviorSubject<number | null>(null);
 
-    public constructor() {
+    private meetingSubject = new BehaviorSubject<ViewMeeting | null>(null);
+    private meetingSubcription: Subscription;
+
+    protected modelSubscription: ModelSubscription | null = null;
+
+    public constructor(private repo: MeetingRepositoryService, private autoupdateService: AutoupdateService) {
+        this.meetingIdSubject.subscribe(id => {
+            this.closeModelSubscription();
+            if (id) {
+                this.requestModel();
+                const meetingObservable = this.repo.getViewModelObservable(id);
+
+                if (this.meetingSubcription) {
+                    this.meetingSubcription.unsubscribe();
+                }
+                this.meetingSubcription = meetingObservable.subscribe(meeting => {
```

Comment:
You can merge this line and line 27 -> `this.meetingSubcription = this.repo.getViewModelObservable(id).subscribe(meeting => {`